deep_quoridor/src/agents/alphazero/nn_evaluator.py

```diff
@@ -63,18 +63,27 @@ def evaluate(self, game: Quoridor):
         assert np.any(policy_masked > 0), "Policy is all zeros"
         assert np.isfinite(policy_masked).all() and np.isfinite(value), "Policy or value is non-finite"
 
+        # If the game was originally rotated, rotate the resulting back to player 2's perspective
+        if is_board_rotated:
+            policy_masked = policy_masked[self.rotated_action_mapping]
+
         return value, policy_masked
 
+    def rotate_game_for_player_two(self, game: Quoridor):
```

Comment:
The name of this function is a bit misleading - given the name, I would expect it to always rotate the board to player two's perspective. Maybe call it `orient_game_for_current_player()`?

Reply:
Also (I realize Im being a bit hypocritical here since i didnt document the code well originally), future us would probably appreciate a doc string for this function. Something like:
    """
    Rotates the game so that the current player's goal is the row with the largest index.

    This makes it easier for the neural network to learn, since it doesn't need to
    understand that one player wants to move up the board and the other wants to move down it. If the current player is Player.ONE, this is a no-op since the board 
    is already rotated correctly for it, and the same game instance is returned. If
    the current player is Player.TWO, the returned game is a copy of the original game
    which has beeen rotated appropriately.
    """